### PR TITLE
Extract .pot files

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

File: src/ussd_health_worker.js [hunk 1]
```diff
@@ -20,52 +20,90 @@ go.app = function() {
...
+                $("Sorry not a valid input. Would they like to receive additional messages about HIV?"),
+            "state_end_thank_you":
+                $("Sorry not a valid input. Thank you. The woman's FamilyConnect ID is {{health_id}}. They will now start receiving messages"),
         };
 
         get_error_text = function(name) {
-            return errors[name] || "Sorry not a valid input. " + questions[name];
+
+            return errors[name];
+            // return errors[name] || questions[name] + $("Sorry not a valid input. ")();
```
Comment (imsickofmaps) [inline src/ussd_health_worker.js]: Commented out - remove or keep?

File: src/ussd_health_worker.js [hunk 2]
```diff
@@ -256,12 +296,12 @@ go.app = function() {
...
         self.add('state_household_head_name', function(name) {
             return new FreeText(name, {
-                question: $(questions[name]),
+                question: questions[name],
                 check: function(content) {
                     if (go.utils.is_valid_name(content, 1, 150)) {
                         return null;  // vumi expects null or undefined if check passes
                     } else {
-                        return $(get_error_text(name));
+                        return get_error_text(name);
```
Comment (imsickofmaps) [inline src/ussd_health_worker.js]: why are some error lookups `errors[name];` and some `get_error_text(name);`?

**Comment by gsvr** [comment]
@imsickofmaps Please review.

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**